lessons/models.py

```diff
@@ -62,6 +62,22 @@ class Meta:
         ordering = ["word"]
         verbose_name_plural = "vocab words"
 
+    @property
+    def i18n_key(self):
+        return slugify(self.get_untranslated_field('word'))
```

Comment:
is this guaranteed to be unique?

Reply:
Good call — added a unique constraint in 43e284886e9bd7185d4729419ed1ca05848055bf. But I need to work with @mrjoshida on getting the existing data to not have duplicates:

```py
In [32]: Vocab.objects.filter(word="Algorithm")
Out[32]: [
  <Vocab: Algorithm: A list of steps that you can follow to finish a task.>,
  <Vocab: Algorithm: A list of steps to finish a task. A set of instructions that can be performed with or without a computer. For example, the collection of steps to make a peanut butter and jelly sandwich is an algorithm.>,
  <Vocab: Algorithm: A list of steps to finish a task. >
]
```

Reply:
Ok, we've removed all duplicates! I pulled down the DB locally and verified the new migration succeeds with the most recent data.